amivapi/tests/events/test_queue.py

```diff
@@ -55,6 +55,15 @@ def test_fcfs_users_get_auto_accepted(self):
                                     status_code=200).json
         self.assertTrue(user2_signup['accepted'])
 
+        # post accepted signup as admin
```

Comment:
It is not clear that this is tested as an admin, I would maybe not use the `webtestnoauth` here.

Additionally, please also test the other direction, i.e. that a non-admin *cannot* set `accepted: True`.